Resources/Prototypes/_DV/Entities/Objects/Specific/syndicate.yml

```diff
@@ -7,3 +7,26 @@
   - type: Store
     balance:
       Telecrystal: 45
+  - type: Tag
+    tags:
+    - NukeOpsUplink
+
+- type: entity
+  parent: BaseUplinkRadio40TC
+  id: CommanderUplinkRadio45TC
+  suffix: 45 TC, NukeOps
```

Comment:
include command and agent in the suffix or there are 3 uplinks with the same suffix